pkg/apis/garden/validation/validation_test.go

```diff
@@ -2800,6 +2804,17 @@ var _ = Describe("validation", func() {
 				}))
 			})
 
+			It("should forbid very frequent backup configuration", func() {
+				shoot.Spec.Backup = invalidFrequentBackup
+
+				errorList := ValidateShoot(shoot)
+				Expect(errorList).To(ConsistOfFields(Fields{
+					"Type":   Equal(field.ErrorTypeInvalid),
+					"Field":  Equal("spec.backup.schedule"),
+					"Detail": Equal(fmt.Sprintf("backup schedule frequency can not be so high, minimum frequency is once every %s", garden.MinimumETCDFullBackupTimeInterval)),
```

Comment:
s/high/low/g

Reply:
@swapnilgm  here I am not talking about the value, rather the frequency of backup, the lower the interval, the more frequent the backups will happen.

Reply:
Ahh ok. All right then.